src/domain/challenge/common/JourneyAboutDialog/JourneyAboutDialog.tsx

```diff
@@ -0,0 +1,235 @@
+import { gutters } from '../../../../core/ui/grid/utils';
+import { Box, BoxProps } from '@mui/material';
+import React, { ReactNode, useMemo, useState } from 'react';
+import { PageTitle, Tagline } from '../../../../core/ui/typography';
+import { HubOutlined } from '@mui/icons-material';
+import Gutters from '../../../../core/ui/grid/Gutters';
+import PageContentColumn from '../../../../core/ui/content/PageContentColumn';
+import PageContentBlock from '../../../../core/ui/content/PageContentBlock';
+import PageContentBlockHeader from '../../../../core/ui/content/PageContentBlockHeader';
+import WrapperMarkdown from '../../../../core/ui/markdown/WrapperMarkdown';
+import EntityDashboardLeadsSection from '../../../community/community/EntityDashboardLeadsSection/EntityDashboardLeadsSection';
+import { JourneyTypeName } from '../../JourneyTypeName';
+import { useTranslation } from 'react-i18next';
+import { EntityDashboardLeads } from '../../../community/community/EntityDashboardContributorsSection/Types';
+import ContactLeadsButton from '../../../community/community/ContactLeadsButton/ContactLeadsButton';
+import {
+  DirectMessageDialog,
+  MessageReceiverChipData,
+} from '../../../communication/messaging/DirectMessaging/DirectMessageDialog';
+import ActivityView from '../../../platform/metrics/views/MetricsView';
+import { ChallengeIcon } from '../../challenge/icon/ChallengeIcon';
+import { OpportunityIcon } from '../../opportunity/icon/OpportunityIcon';
+import { Metric } from '../../../platform/metrics/utils/getMetricCount';
+import useMetricsItems from '../../../platform/metrics/utils/useMetricsItems';
+import HubMetrics from '../../hub/Metrics/HubMetrics';
+import ChallengeMetrics from '../../challenge/utils/useChallengeMetricsItems';
+import OpportunityMetrics from '../../opportunity/utils/useOpportunityMetricsItems';
+import { Theme } from '@mui/material/styles';
+import useCurrentBreakpoint from '../../../../core/ui/utils/useCurrentBreakpoint';
+import PageContentBlockSeamless from '../../../../core/ui/content/PageContentBlockSeamless';
+
+export interface JourneyAboutDialogProps extends EntityDashboardLeads {
+  open: boolean;
+  journeyTypeName: JourneyTypeName;
+  displayName: ReactNode;
+  tagline: ReactNode;
+  ribbon?: ReactNode;
+  startButton?: ReactNode;
+  endButton?: ReactNode;
+  sendMessageToCommunityLeads: (message: string) => Promise<void>;
+  metrics: Metric[] | undefined;
+  description: string | undefined;
+  background: string | undefined;
+  who: string | undefined;
+  impact: string | undefined;
+  loading: boolean;
+}
+
+interface DialogHeaderItemProps extends BoxProps {
+  align?: 'start' | 'end' | 'center';
+}
+
+const DialogHeaderItem = ({ align = 'center', ...props }: DialogHeaderItemProps) => {
+  return <Box {...props} flexGrow={1} display="flex" justifyContent={align} alignItems="center" gap={gutters()} />;
+};
+
+const getJourneyIconComponent = (journeyTypeName: JourneyTypeName) => {
```

Comment:
we have this also in: src/domain/shared/components/JourneyIcon/JourneyIcon.tsx
I see now that it is from the old code but if you can use the common one, please...